web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx

```diff
@@ -107,8 +110,8 @@ const properties: ExternalCallFilterProperties = {
   outOfScheduleEnabled: {
     label: _('Out of schedule enabled'),
     enum: {
-      '0': _('No'),
-      '1': _('Yes'),
+      '0': _('Disabled'),
+      '1': _('Enabled'),
```

Comment:
If you update this translations, you must also update holidayEnabled field. Those fields are used to toggle the rest of the fields of its section.

Reply:
I rather revert it. The thing is that outOfScheduleTargetType has different labels in ExternalCallFilters and ExternalCallFiltersResidential

Reply:
Ok, revert it and I'll fix that in #2277